cloud/aws/integrations-aws.tf

```diff
@@ -1,12 +1,42 @@
-resource "signalfx_aws_external_integration" "sfx_integration_external" {
-  name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+locals {
```

Comment:
not mandatory, but we like to extract locals constants to a locals.tf file

Reply:
done